### PR TITLE
A possible fix for orca #133 by changing how paths are parsed; …

### PR DESCRIPTION
**Pull request by ibaldin**
This changes how the path is parsed by skipping additional element when stitchport is detected. Should not affect any other situation. 

Needs cleaning up -  I just want consensus on the solution. 

Regular paths go node-link-node-link-node. Stitchport paths go port-node-link-node-link-node. I need to skip links as they are not shown.

### CODE REVIEW

**Comment by YufengXin** [comment]
Since it doesn't affect else, should deploy and give it a try.

File: pom.xml [hunk 1]
```diff
@@ -302,7 +302,7 @@
 		<dependency>
 			<groupId>orca</groupId>
 			<artifactId>ndl</artifactId>
-			<version>5.1.5-SNAPSHOT</version>
+			<version>5.2.2-SNAPSHOT</version>
```
Comment (hinchliff) [inline pom.xml]: This isn't necessarily wrong in this particular case, but by default / in general, Flukes should be using Release artifacts from Orca.

File: src/main/java/orca/flukes/ndl/ManifestLoader.java [hunk 2]
```diff
@@ -827,25 +827,34 @@ public void ndlNetworkConnectionPath(Resource c, OntModel m,
...
 					// only take nodes, skip links on the path
 					pIter.next();
-					if (!pIter.hasNext())
+					if (!pIter.hasNext()) {
 						break;
+					}
 					Resource second = pIter.next();
 					OrcaNode firstNode = nodes.get(getTrueName(first));
 					OrcaNode secondNode = nodes.get(getTrueName(second));
+					System.out.println("first " + getTrueName(first) + " and second " + getTrueName(second) + " " + firstNode + "/" + secondNode);
```
Comment (hinchliff) [inline src/main/java/orca/flukes/ndl/ManifestLoader.java]: Should probably change this to a logger statement?

File: pom.xml (hunk 1 above)
Comment (ibaldin) [reply pom.xml]: I'll fix that

File: src/main/java/orca/flukes/ndl/ManifestLoader.java (hunk 2 above)
Comment (ibaldin) [reply src/main/java/orca/flukes/ndl/ManifestLoader.java]: I'll get rid of it. This is what I meant by 'cleaning up'

**Pull request merged**